OpenSim/Common/ComponentList.h

```diff
@@ -168,6 +168,10 @@ class ComponentListIterator :
     bool operator==(const ComponentListIterator& iter) const {
         return _node == &*iter;
     }
+    /// Check for equality using method rather than operator to support scripting
+    bool equals(const ComponentListIterator& iter) const {
```

Comment:
@sherm1 My understanding of the syntax is that the method is just invoking the operator (the operator came first), please let me know if that's not the case or suggest the correct syntax. Both have been used and behave as expected but that maybe just a coincidence.